src/pages/Vaults/VaultsOverview/components/CreateVaultWizard/CreateVaultWizard.tsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from 'react';
+
+import { VaultsTableRow } from '../VaultsTable';
+import DespositCollateralStep from './DespositCollateralStep';
+import DisclaimerStep from './DisclaimerStep';
+import { Steps } from './Step';
+import VaultCreatedStep from './VaultCreatedStep';
+
+type Props = {
+  vault?: VaultsTableRow;
+};
+
+type CreateVaultWizardProps = Props;
+
+const CreateVaultWizard = ({ vault }: CreateVaultWizardProps): JSX.Element | null => {
+  const [step, setStep] = useState<Steps>(1);
+
+  const handleNextStep = () => setStep((s) => (s + 1) as Steps);
```

Comment:
Not a blocker, but can you give some thought to how we would abstract this out to a generic multi-step modal? We don't want to do anything now, but if we need to do the same elsewhere on the app we want to make it as easy as possible to reuse this code.

Reply:
I will add a todo here for that.